cpp/farm_ng/core/logging/format.h

```diff
@@ -23,21 +23,11 @@
 #endif
 
 namespace farm_ng {
-inline std::function<void(std::string const &)> &getLogLineFunction() {
```

Comment:
@strasdat I moved this to a shared library as this singleton needs to be set across shared library boundaries in normal usecases and the inline static function will not always behave as expected.

Reply:
Sounds good. I don't think it is critical to keep farm-ng-core header only.